onnxruntime/test/opaque_api/test_opaque_api.cc

```diff
@@ -19,6 +19,8 @@
 using namespace ONNX_NAMESPACE;
 using namespace onnxruntime::common;
 
+const OrtApi* Ort::g_api = OrtGetApi(ORT_API_VERSION);
```

Comment:
Does every executable have to do it? Can this hit global object order of creation problem?

Reply:
It can be done inside the functions that use it. This was just the simplest way to modify the current usage of the APIs. The only order problem would be if someone calls Ort* APIs in another static before this one gets initialized.